home-page

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Binabox NFT Collection</title>
+</head>
+<body>
+
+    <h2>Binabox</h2>
+    <p><a href="#">Home</a></p>
+    <p><a href="#">About</a></p>
+    <p><a href="#">Roadmap</a></p>
+    <p><a href="#">Collection</a></p>
+    <p><a href="#">FAQs</a></p>
+    <p><a href="#">Pages</a></p>
+
+    <br>
+
+    <p><a href="#">Discord</a> | <a href="#">Connect</a></p>
+
+    <h1>WITH BINABOX EXPLORE NFT COLLECTION</h1>
```

Comment:
I see that you have h1 present! It's very cool that the sequence of titles is followed. Now let's try the order, that is, usually the information goes from top to bottom and the main heading (h1) is higher than h2, and h2 will be higher than h3 in the markup.

It's like a "Category", then a "subcategory", then something else less significant on the page

Reply:
can i put them in the order that i have to and then use css to put them where they have to go, like on the figma preview??